### PR TITLE
[ru] Translation conventions

### PR DESCRIPTION
**Pull request by dios-oleg**
translation

### CODE REVIEW

**Comment by dios-oleg** [comment]
corrected some grammatical errors

**Comment by dios-oleg** [comment]
corrected grammatical errors

**Comment by Leo240** [comment]
Thanks for your help. I have been shocked how I could do so many mistakes.

File: ru/intro/conventions.rst [hunk 1]
```diff
@@ -47,12 +47,12 @@ cледовательно ``/article-categories/view-all`` это коррект
...
-Для большей информации о URL-адресаах CakePHP и обработке параметров, смотрите
-:ref:`routes-configuration`.
+Для более подробной информации о URL-адресах CakePHP и обработке параметров,
+смотрите :ref:`routes-configuration`.
 
 .. _file-and-classname-conventions:
 
-Соглашения об именах классов и файлов
+Соглашения об Наименовании классов и Файлов
 =====================================
```
Comment (cake17) [inline ru/intro/conventions.rst]: The underline `===` should be as long as the title

File: ru/intro/conventions.rst [hunk 2]
```diff
@@ -79,54 +79,54 @@ cледовательно ``/article-categories/view-all`` это коррект
...
 В дополнение к использованию в качестве первичого ключа обычного
-автоинкрементируемого поля, Вы также можете использовать в этих целях поля
+автоинкрементного поля, Вы также можете использовать в этих целях поля
 UUID. CakePHP создаст уникальный 36-значный UUID
 (:php:meth:`Cake\\Utility\\Text::uuid()`), всякий раз когда Вы вызовете метод
 ``Table::save()``.
 
-Соглашения вида
+Соглашения Вида (Представления)
 ===============
```
Comment (cake17) [inline ru/intro/conventions.rst]: Same here

Comment (dios-oleg) [inline ru/intro/conventions.rst]: yes, it's synonymous, but the word "Вид"  in this context is inappropriate to use.

https://ru.wikipedia.org/wiki/Model-View-Controller

Comment (Leo240) [inline ru/intro/conventions.rst]: It means not that the words are same, but that underline must be as long as title

Comment (cake17) [inline ru/intro/conventions.rst]: @Leo240 yes that's what I meant. Nice work on the russian translation by the way ! 👍

**Pull request merged**